src/project-manager.ts

```diff
@@ -316,12 +315,8 @@ export class ProjectManager implements Disposable {
 		if (observable) {
 			return observable;
 		}
-		const parts = url.parse(uri);
-		if (!parts.pathname) {
-			return Observable.throw(new Error(`Invalid URI ${uri}`));
-		}
 		// TypeScript works with file paths, not URIs
-		const filePath = parts.pathname.split('/').map(decodeURIComponent).join('/');
+		const filePath = util.uri2path(uri);
```

Comment:
Oh, right! makes sense. I think we should match the path for a drive letter + colon (colon is reserved on Unix as a PATH seperator) and strip the leading slash in that case.